packages/mobile/src/screens/create-chat-blast-screen/ChatBlastSelectAudienceFields.tsx

```diff
@@ -63,10 +75,12 @@ const FollowersMessageField = () => {
   const { data: user } = useGetCurrentUser()
   const [{ value: targetAudience }] = useField(TARGET_AUDIENCE_FIELD)
   const isSelected = targetAudience === ChatBlastAudience.FOLLOWERS
+  const isDisabled = user?.follower_count === 0
```

Comment:
wdym? `user && user.follower_count === 0`? i think `undefined === 0` comes out to false?

Reply:
yeah so disabled would be false. Wouldn't we want `isDisabled: true` when there's no user also?

Again, I don't think it's possible to get here without a user though, since it's behind the CTA?

Reply:
u right i was inverted